ext/win32ole/win32ole.c

```diff
@@ -687,10 +687,19 @@ ole_cp2encoding(UINT cp)
         }
     }
 
-    enc_name = rb_sprintf("CP%d", cp);
-    idx = rb_enc_find_index(enc_cstr = StringValueCStr(enc_name));
+    switch(cp) {
+      case 20932:
+        enc_cstr = "eucJP-ms";
```

Comment:
Is mapping CR20932 to `eucJP-ms` correct? It seems so from the mapping in ole_encoding2cp and the comments in tests (`WIN32OLE.codepage = 20932 # MS EUC-JP`)